cypress/e2e/aboutpage.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe("About page tests", () => {
+  beforeEach(() => {
+    cy.visit("/about",  { timeout: 30000 });
+  });
+
+  it("About header renders", () => {
+    cy.get('[data-cy="about-header-header"]')
+      .should("be.visible")
+      .and("contain", "Transnational Job Listing Channel");
+    cy.get('[data-cy="about-header-body"]')
+      .should("be.visible")
+      .and("contain", "So many jobs available, all you have to do is keep up with  our posts. Check below for recent job openings.");
+  });
+
+  it("About banner renders", () => {
+    cy.get('[data-cy="about-banner-header"]')
+      .should("be.visible")
+      .and("contain", "Have a question?");
+    cy.get('[data-cy="about-banner-body"]')
+      .should("be.visible")
+      .and("contain", "If you have any questions, please contact us");
+    cy.get('[data-cy="about-banner-button"]')
+      .within(() => {
+        cy.validateLink('Contact Us', '/contact');
+      });
+  });
```

Comment:
What's missing here is a test of the content on the right hand side, which only appears on wider viewports:

<img width="817" alt="Screenshot 2023-10-01 at 11 59 01 AM" src="https://github.com/TechIsHiring/techishiring-website/assets/8340719/62fc7102-2d7f-48c1-af5d-9992bc4ad03b">

I'd suggest we change the default viewport size for `e2e` in `cypress.config.ts` to 1400, like this:

```js
  e2e: {
    baseUrl: 'http://localhost:3000',
    viewportWidth: 1400,
    ...
```

That will make all tests of "desktop layout" be more representative.

Reply:
Added tests for the rhs content